src/pages/restaurant/components/menu/helpers/getDishById.js

```diff
@@ -0,0 +1,11 @@
+function getDishById(id, dishes) {
+  for (const items of Object.values(dishes)) {
+    const dish = items.find((item) => item.id === id);
+    if (dish) {
```

Comment:
check once does this check required because if there will be no dish with the required id it will return undefined

Reply:
Yes because here items is an array of objects